### PR TITLE
add markdown

### PR DESCRIPTION
**Pull request by gro1m**

### CODE REVIEW

File: pycobertura/reporters.py [hunk 1]
```diff
@@ -58,6 +58,11 @@ def generate(self):
         lines = self.get_report_lines()
         return tabulate(lines, headers=headers_with_missing)
 
+class MarkdownReporter(Reporter):
```
Comment (aconrad) [inline pycobertura/reporters.py:61]: We'll need extra space to get the tests to pass, I think.

```suggestion

class MarkdownReporter(Reporter):
```

**Comment by gro1m** [comment]
Hi @aconrad 
I added the README, the tests (everything green now) and the CHANGES :)

File: CHANGES.md [hunk 2]
```diff
@@ -13,6 +13,9 @@
   * change datastructure of coverage information from `namedtuple` (immutable) to `dictionary` (mutable) in `reporters.py`. Adjusted `html-delta.jinja2`, `html.jinja2` and `filters.py` accordingly. This change in datastructure leads to a more compact and more readable code.
   * `pycobertura show` now includes json format
   * `pycobertura diff` now includes json format
+  * `pycobertura show` now includes markdown format
+  * `pycobertura diff` now includes markdown format
```
Comment (aconrad) [inline CHANGES.md:17]: Can we combine these lines to make it more digestible for the reader?

**Review by aconrad — APPROVED** [review]
This is great! 😍 I'll merge this once we make the CHANGE file more concise (see my comment).

**Comment by aconrad** [comment]
💃🕺

**Pull request merged**